ui/app/components/RootComponent/RootComponent.js

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react';
+import Header from '../HeaderComponent';
```

Comment:
See if you can modify imports of local files to use paths relative to root of application rather than relative paths. Makes the import and module location readable, less prone to copy-paste failures. Applies throughout the PR.